src/Helpers/OrderBuilderTrait.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+namespace GingerPluginSdk\Helpers;
+
+use GingerPluginSdk\Entities\Order;
+use GingerPluginSdk\Response\GingerHTTPResponse;
+use GingerPluginSdk\Response\GingerHTTPResponseBody;
+use JetBrains\PhpStorm\ArrayShape;
+
+trait OrderBuilderTrait
```

Comment:
The OrderBuilderTrait is only used once, why not inline the implementation in the Client class?

Reply:
I want to use Client for more than only OrderBuilder methods, in this way, for me, the best way is to handle Client realization with only related to all dependencies methods and using Traits to separate functionality. 

Could you clarify your vision in this case?